### PR TITLE
Update StormForge Optimize Live dashboard to new version

### PR DESCRIPTION
**Pull request by reidmv**
### What does this PR do?

This commit updates the out-of-the-box (OOTB) StormForge Optimize Live dashboard to present much the same view in Datadog as we have designed for our in-product visualizations.

### Motivation

I work for StormForge. We are working on improving the UX for our StormForge+Datadog integration users.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

This is a significant change and improvement over our initial dashboard design. It is effectively a rewrite of the dashboard, in terms of scope. If you have any questions or would like assistance conducting a review feel free to reach out.

### CODE REVIEW

File: stormforge/assets/dashboards/stormforge_overview.json [hunk 1]
```diff
@@ -1,545 +1,1035 @@
...
...rge-optimize-live-works/) applies machine learning to your observability metrics to make real-time recommendations to resource requests for any deployment running in Kubernetes.\n\nWith StormForge Optimize Live, you can:\n\n* Improve resource efficiency\n* Leverage existing observability data\n* Reduce risk of performance issues\n* Achieve fast time to value\n* Deploy recommendations automatically or with approval\n\nThe Datadog Workload Overview dashboard lets you preview recommendations made by Optimize Live on a _single workload_ basis. To get started:\n\n1. Ensure Optimize Live is [installed](https://docs.stormforge.io/optimize-live/getting-started/install/) and [configured](https://docs.stormforge.io/optimize-live/getting-started/configuration/)\n2. Select a single value for each of the dashboard's template variables\n3. Review the Recommendation Summary group to understand the overall impact of the recommendation (inclusive of replica scale-up and scale-down events)\n4. Review the Details group to understand the impact of the recommendation on an average container (replica) in the workload\n5. When ready, use Optimize Live to start automatically applying the recommendations",
```
Comment (emarsha94) [inline stormforge/assets/dashboards/stormforge_overview.json:3]: We typically like to have the overview in dashboards be a couple of sentences. Could you include a couple of sentences on the value of the dashboard?

**Review by emarsha94 — CHANGES_REQUESTED** [review]
Let me know if you have any questions on the comments/suggestions!

File: stormforge/assets/dashboards/stormforge_overview.json (hunk 1 above)
Comment (reidmv) [reply stormforge/assets/dashboards/stormforge_overview.json:3]: I'll make and commit a suggested change here in the comment for ease of documentation. Let me know if this is better.

```suggestion
  "description": "StormForge Optimize Live applies machine learning to your observability metrics to make real-time recommendations to resource requests for any deployment running in Kubernetes. The Workload Overview dashboard lets you preview the recommendations made by Optimize Live, and the projected impact of applying those recommendations. The dashboard is designed to show recommendations and impact on a per-workload basis.",
```

**Comment by reidmv** [comment]
@emarsha94 Changes applied. Ready for re-review.

**Review by emarsha94 — APPROVED** [review]
Thanks for making the update! Approving

**Pull request merged**